pkgs/tools/text/groff/default.nix

```diff
@@ -14,31 +14,15 @@
 
 stdenv.mkDerivation rec {
   pname = "groff";
-  version = "1.22.4";
+  version = "1.23.0";
 
   src = fetchurl {
     url = "mirror://gnu/groff/${pname}-${version}.tar.gz";
-    sha256 = "14q2mldnr1vx0l9lqp9v2f6iww24gj28iyh4j2211hyynx67p3p7";
+    hash = "sha256-a5dX9ZK3UYtJAutq9+VFcL3Mujeocf3bLTCuOGNRHBM=";
   };
 
   outputs = [ "out" "man" "doc" "info" "perl" ];
 
-  # Parallel build is failing for missing depends. Known upstream as:
-  #   https://savannah.gnu.org/bugs/?62084
-  #   fixed, planned release: 1.23.0
-  enableParallelBuilding = false;
```

Comment:
Consider enabling parallelism here with `enableParallelBuilding = true;`. `groff` is a good example of a package where it has a visible benefit: on my machines parallel build takes 20 seconds (mainly in `./configure`) while sequential takes 52s.

Reply:
I assumed that the explicit `enableParallelBuilding = false;` meant that parallel building is enabled by default; this PR is now update accordingly, thanks @trofi.